### PR TITLE
Added clarification on how this role is intended to be run

### PR DESCRIPTION
**Pull request by qbunt**
Also added a gitignore. Thanks so much for making this thing, happy to discuss any changes here. This was a real help yesterday for a staging site we're building. Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,16 @@ roles:
 
 Follow the example above to get started. if the `htpasswd` key is removed from the environment file, the protection will be removed on the next deploy.
 
+Adding / Removing Basic Authentication
+-------------------------------
```
Comment (louim) [inline README.md]: Can you make it so the dashes match the length of the title to stay consistent with the other titles?

File: README.md [hunk 2]
```diff
@@ -73,6 +73,16 @@ roles:
...
+**To Add**: Run the Trellis command to set up your previously configured remote server: `ansible-playbook server.yml -e env=<environment>`
+**To Remove**: Remove the following `htpasswd` block:
+<pre><b>htpasswd:
+           name: user
+           password: secret</b>
+</pre>
+in the `wordpress_sites` dict set, and redeploy via: `./deploy.sh <environment> <site name>`.
+
```
Comment (louim) [inline README.md:86]: The remove will need to run the same command that the "add", (`ansible-playbook server.yml -e env=<environment>`). The `deploy.sh` would not work to disable it.

File: README.md [hunk 3]
```diff
@@ -73,6 +73,16 @@ roles:
 
 Follow the example above to get started. if the `htpasswd` key is removed from the environment file, the protection will be removed on the next deploy.
 
+Adding / Removing Basic Authentication
+-------------------------------
+**To Add**: Run the Trellis command to set up your previously configured remote server: `ansible-playbook server.yml -e env=<environment>`
+**To Remove**: Remove the following `htpasswd` block:
+<pre><b>htpasswd:
+           name: user
```
Comment (louim) [inline README.md]: Can you make the indentation consistent with the first example (two space)? ex:

```
htpasswd:
  name: user
  password: secret
```

**Comment by louim** [comment]
Thanks for the PR! I would rather have a smaller `.gitignore` with only

```
.DS_Store
```

or no `.gitignore` at all since those ignores are often project / language specific.

I also left some comment on the readme :smile:

File: README.md (hunk 1 above)
Comment (qbunt) [inline README.md]: Sure thing!

File: README.md (hunk 2 above)
Comment (qbunt) [inline README.md:86]: Sure, so the previous paragraph:

> Follow the example above to get started. if the `htpasswd` key is removed from the environment file, the protection will be removed on the next deploy.

Is that not accurate? Maybe I'm misunderstanding?

File: README.md (hunk 3 above)
Comment (qbunt) [inline README.md]: Sure thing!

**Comment by qbunt** [comment]
Made all the changes you mentioned, I did remove one line that confused me a bit. With Trellis, I took deploy to mean a run of the `deploy.sh` script. Let me know if I misunderstood how that was intended. Thanks for reviewing!

File: README.md (hunk 2 above)
Comment (louim) [inline README.md:86]: You are right. It should read "on the next server provision". 

The `deploy.sh` script just run the deploy.yml playbook.

**Comment by louim** [comment]
Thank you very much! :+1:

**Pull request merged**